ui/v2.5/src/locales/en-GB.json

```diff
@@ -237,7 +237,9 @@
       "sqlite_location": "File location for the SQLite database (requires restart)",
       "video_ext_desc": "Comma-delimited list of file extensions that will be identified as videos.",
       "video_ext_head": "Video Extensions",
-      "video_head": "Video"
+      "video_head": "Video",
+      "include_audio_head": "Include audio",
+      "include_audio_desc": "Includes audio stream when generating previews."
```

Comment:
What are these? I don't see any reference in the tsx files.

Reply:
Huh, it's supposed to be under the generated content settings. I thought I replaced the string in there. I'll fix it when I'm home, or you could make a commit. Either is fine by me.